doc/man1/openssl-pkcs12.pod.in

```diff
@@ -341,10 +343,18 @@ key and the certificates are encrypted using AES-256-CBC unless
 the '-legacy' option is used. If '-descert' is used with the '-legacy'
 then both, the private key and the certificates are encrypted using triple DES.
 
+=item B<-pbmac1> I<digest>
+
+Use PBMAC1 for MAC protection of the PKCS#12 file.
+
 =item B<-macalg> I<digest>
 
 Specify the MAC digest algorithm. If not included SHA256 will be used.
 
+=item B<-pbmac1_pbkdf2_kdf_md> I<digest>
+
+Specify the PBMAC1 KDF digest algorithm. If not included SHA256 will be used.
```

Comment:
`specify the PBKDF2 KDF digest algorithm`

Reply:
Fixed